superset-frontend/src/views/CRUD/data/database/DatabaseModal.tsx

```diff
@@ -203,16 +332,322 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
           </StyledInputContainer>
         </Tabs.TabPane>
         <Tabs.TabPane tab={<span>{t('Performance')}</span>} key="2">
-          Performance Form Data
+          <StyledInputContainer>
+            <div className="label">{t('Chart Cache Timeout')}</div>
+            <div className="input-container">
+              <input
+                type="number"
+                name="cache_timeout"
+                value={db ? db.cache_timeout || '' : ''}
+                placeholder={t('Chart Cache Timeout')}
+                onChange={onInputChange}
+              />
+            </div>
+            <div className="helper">
+              {t(
+                'Duration (in seconds) of the caching timeout for charts of this database.',
+              )}
+              {t(' A timeout of 0 indicates that the cache never expires.')}
+              {t(' Note this defaults to the global timeout if undefined.')}
+            </div>
+          </StyledInputContainer>
+          <StyledInputContainer>
+            <div className="input-container">
+              <IndeterminateCheckbox
+                id="allow_run_async"
+                indeterminate={false}
+                checked={db ? !!db.allow_run_async : false}
+                onChange={onInputChange}
+              />
+              <div>{t('Asynchronous Query Execution')}</div>
+              <InfoTooltipWithTrigger
+                label="aqe"
+                tooltip={
+                  t(
+                    'Operate the database in asynchronous mode, meaning that the queries ',
+                  ) +
+                  t(
+                    'are executed on remote workers as opposed to on the web server itself. ',
+                  ) +
+                  t(
+                    'This assumes that you have a Celery worker setup as well as a results ',
+                  ) +
+                  t(
+                    'backend. Refer to the installation docs for more information.',
```

Comment:
I'm not sure translations work like this. Each of these will be extracted as separate strings and not all language adhere to the same grammar rules. I think each of these sentences needs to be wrapped in a single `t` so they're extracted as a single blob, I think something like 

```js
t('............very long string ...............' +
   '.............another long string ..........' + 
   ... 
)
``` 

should work.